### PR TITLE
chore(core): migrate greenhouse dashboard

### PR DESCRIPTION
**Pull request by andypf**
# Summary

This PR migrates the Greenhouse Dashboard along with all dependent applications to the Juno framework under the apps/greenhouse directory. Dependent apps are now organized under apps/greenhouse/apps to streamline the structure and maintain consistency.

# Changes Made

- creates apps/greenhouse
- configures greenhouse and all dependent apps to meet juno repo guide lines
- extends the package.json
- fixes all lint and format issues

# Testing Instructions

1. `npm i`
2. `npm run test`

# Checklist

- [x] I have performed a self-review of my code.
- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] I have made corresponding changes to the documentation (if applicable).
- [x] My changes generate no new warnings or errors.

### CODE REVIEW

**Review by hgw77 — APPROVED** [review]

File: apps/greenhouse/apps/types/package-lock.json
Comment (ArtieReus) [inline apps/greenhouse/apps/types/package-lock.json:1]: remove

File: apps/greenhouse/apps/types/package.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cloudoperators/juno-app-greenhouse-types",
```
Comment (ArtieReus) [inline apps/greenhouse/apps/types/package.json:2]: we should add at least private true so it does not get accidentally published

File: apps/greenhouse/apps/utils/package-lock.json
Comment (ArtieReus) [inline apps/greenhouse/apps/utils/package-lock.json:1]: remove

File: apps/greenhouse/apps/utils/package.json [hunk 2]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "@cloudoperators/juno-app-greenhouse-utils",
```
Comment (ArtieReus) [inline apps/greenhouse/apps/utils/package.json:2]: we should add at least private true so it does not get accidentally published

File: apps/greenhouse/package-lock.json
Comment (ArtieReus) [inline apps/greenhouse/package-lock.json:1]: remove

File: apps/greenhouse/package.json [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+  "name": "@cloudoperators/juno-app-greenhouse",
+  "version": "0.1.27",
+  "orgAdminVersion": "latest",
+  "author": "UI-Team",
+  "contributors": [
+    "Andreas Pfau",
+    "Arturo Reuschenbach Puncernau",
+    "Esther Schmitz"
+  ],
+  "repository": "https://github.com/cloudoperators/greenhouse/tree/main/ui/dashboard",
```
Comment (ArtieReus) [inline apps/greenhouse/package.json:11]: wrong repo

File: apps/greenhouse/package.json [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+  "license": "Apache-2.0",
+  "source": "src/index.js",
+  "module": "build/index.js",
+  "private": false,
```
Comment (ArtieReus) [inline apps/greenhouse/package.json:15]: private true

File: apps/greenhouse/package.json [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+    "util": "^0.12.4",
+    "zustand": "4.5.5"
+  },
+  "scripts": {
+    "lint": "eslint",
+    "test": "jest",
+    "dev": "NODE_ENV=development node esbuild/config.js --serve --watch",
+    "build": "NODE_ENV=production node esbuild/config.js"
+  },
+  "peerDependencies": {
```
Comment (ArtieReus) [inline apps/greenhouse/package.json:55]: remove?

File: package-lock.json
Comment (ArtieReus) [inline package-lock.json:1]: remove

**Review by ArtieReus — CHANGES_REQUESTED** [review]

File: apps/greenhouse/apps/cluster-admin/package-lock.json [hunk 6]
```diff
@@ -0,0 +1,11630 @@
+{
```
[the remark this answers is not in the document]
Comment (andypf) [reply apps/greenhouse/apps/cluster-admin/package-lock.json:1]: good point

File: apps/greenhouse/apps/cluster-admin/package.json [hunk 7]
```diff
@@ -0,0 +1,93 @@
...
+  "author": "Services-Team",
+  "contributors": [
+    "Uwe Mayer"
+  ],
+  "repository": "https://github.com/cloudoperators/juno/tree/main/apps/greenhouse/apps/cluster-admin",
+  "appPreview": true,
+  "source": "src/index.js",
+  "module": "build/index.js",
+  "license": "Apache-2.0",
+  "private": false,
```
[the remark this answers is not in the document]
Comment (andypf) [reply apps/greenhouse/apps/cluster-admin/package.json:13]: 👍

File: apps/greenhouse/apps/cluster-admin/package.json [hunk 8]
```diff
@@ -0,0 +1,93 @@
+{
+  "name": "@cloudoperators/juno-app-greenhouse-cluster-admin",
+  "version": "1.6.15",
+  "author": "Services-Team",
```
[the remark this answers is not in the document]
Comment (andypf) [reply apps/greenhouse/apps/cluster-admin/package.json:4]: in the next step we will move those apps into code base of greenhouse. All package.json files will be removed

File: apps/greenhouse/apps/org-admin/package.json [hunk 9]
```diff
@@ -0,0 +1,120 @@
...
+      "version": "latest",
+      "releaseState": "beta"
+    }
+  },
+  "author": "UI-Team",
+  "contributors": [
+    "Arturo Reuschenbach Puncernau",
+    "Tillman Haupt"
+  ],
+  "repository": "https://github.com/cloudoperators/juno/tree/main/apps/greenhouse/apps/org-admin",
```
[the remark this answers is not in the document]
Comment (andypf) [reply apps/greenhouse/apps/org-admin/package.json:33]: hm, it is the correct one!

File: apps/greenhouse/apps/types/package.json (hunk 1 above)
Comment (andypf) [reply apps/greenhouse/apps/types/package.json:2]: 👍

File: apps/greenhouse/apps/utils/package.json (hunk 2 above)
Comment (andypf) [reply apps/greenhouse/apps/utils/package.json:2]: 👍

**Review by ArtieReus — APPROVED** [review]
lgtm

**Review by TilmanHaupt — APPROVED** [review]

**Pull request merged**